src/WebJobs.Script/Diagnostics/OpenTelemetry/OpenTelemetryConfigurationExtensions.cs

```diff
@@ -65,31 +85,22 @@ private static IOpenTelemetryBuilder ConfigureMetrics(this IOpenTelemetryBuilder
                 .AddMeter(HostMetrics.FaasMeterName)
                 .AddView(HostMetrics.FaasInvokeDuration, new ExplicitBucketHistogramConfiguration
                 {
-                    Boundaries = new double[] { 0.005, 0.01, 0.025, 0.05, 0.075, 0.1, 0.25, 0.5, 0.75, 1, 2.5, 5, 7.5, 10 }
+                    Boundaries = new double[] { 0.005, 0.01, 0.025, 0.05, 0.075, 0.1, 0.25, 0.5, 0.75, 1, 2.5, 5, 7.5, 10 },
+                    RecordMinMax = true
                 });
-
-                // Avoid configuring the exporter in placeholder mode, as it will default to sending telemetry to the predefined endpoint. These transmissions will be unsuccessful and create unnecessary noise.
-                if (telemetryMode != TelemetryMode.Placeholder)
-                {
-                    if (enableOtlp)
-                    {
-                        builder.AddOtlpExporter();
-                    }
-                    if (enableAzureMonitor)
-                    {
-                        builder.AddAzureMonitorMetricExporter(opt => ConfigureAzureMonitorOptions(opt, azMonConnectionString, credential));
-                    }
-                }
             });
         }
 
-        private static IOpenTelemetryBuilder ConfigureTracing(this IOpenTelemetryBuilder builder, bool enableOtlp, bool enableAzureMonitor, string azMonConnectionString, TokenCredential credential, TelemetryMode telemetryMode)
+        private static IOpenTelemetryBuilder ConfigureTracing(this IOpenTelemetryBuilder builder)
         {
             return builder.WithTracing(builder =>
             {
                 builder
-                .AddSource("Azure.*")
-                .AddSource("Microsoft.Azure.Webjobs.Extensions.*")
+                .AddSource("Azure.Messaging.ServiceBus.ServiceBusProcessor")
+                .AddSource("Azure.Messaging.EventHubs.EventProcessor")
```

Comment:
Do we have a plan for making this user configurable? Seems like we could have a config section for it.